packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -320,7 +320,7 @@ export class DocumentDeltaConnection
         this.disposeCore(
             false, // socketProtocolError
             createGenericNetworkError(
-                "clientClosingConnection", undefined, { canRetry: true }, { driverVersion }));
+                "Client closing delta connection", { canRetry: true }, { driverVersion }));
```

Comment:
seem like a lot of churn for not much gain

Reply:
the longer string will also negatively affect bundle size

Reply:
One specific goal here is addressing #8474 in which @vladsud pointed out that the short versions are not palatable for developers, compared to more readable (and yes, verbose) sentences.  I am in agreement with that - readable error messages are an important part of the dev experience.  I could open a follow-up issue to estimate or even monitor the impact on bundle size made by error strings (basically all string literals containing a space?).  But my sense is that in general I don't think the delta between sentence v. pascalCaseTerm will be very big, so not worth it.

As for the churn, yeah I feel a little bad. There will be social pain of the canonical phrase/term for particular issues. And in queries, this will work against the goal I have here of effective aggregation, for probably 3-6 months. But I believe in the change, and there's no particular reason now's not a good time to rip off the band-aid.

Reply:
seems fine. i'm not sure how many of these are truly user facing.